src/devices/bluetooth.c

```diff
@@ -95,6 +95,8 @@ static const char * baudConfigCmdForRate(unsigned int baudRate)
                 return "AT+BAUD7";
         case 115200:
                 return "AT+BAUD8";
+        case 230400:
```

Comment:
Oh yeah... I noticed that when I was trying to debug why BT wasn't operational.  I could include that as part of another change if you like.